clang/lib/CodeGen/CGDebugInfo.cpp

```diff
@@ -3886,7 +3886,8 @@ CGDebugInfo::getGlobalVariableForwardDeclaration(const VarDecl *VD) {
   auto Align = getDeclAlignIfRequired(VD, CGM.getContext());
   auto *GV = DBuilder.createTempGlobalVariableFwdDecl(
       DContext, Name, LinkageName, Unit, Line, getOrCreateType(T, Unit),
-      !VD->isExternallyVisible(), nullptr, TemplateParameters, Align);
```

Comment:
Changes in file need the ENABLE_CLASSIC_FLANG guard?

Reply:
Unlike the changes we make in Driver, the optional `flags` field we added to the `DIGlobalVariable` metadata has not caused much problem or confusion historically. Inserting `#ifdef` in all the places where we use the debuginfo metadata APIs will likely create much more verbose delta, but no improvement in clarity.